### ISSUE TITLE
Error: Buffer.write(string, encoding, offset[, length]) is no longer supported

### ISSUE THREAD

**Issue by grantsru** [twilson63/express-couchFs#10]
After upgrading one of our apps to Node v8.x, I am continuously getting an error with file uploads :
```
Error: Buffer.write(string, encoding, offset[, length]) is no longer supported
```
I believe this is an issue with the middleware `express.multipart()`, which has since been separated from the main module and partitioned into another called `multer`. I wanted to document this issue as the couchFs module is currently being used and will require an adjustment.

**Comment by twilson63** [comment]
Cool, I will update it to Node v8 compatibility with express v4 and nano v7

**Comment by grantsru** [comment]
Sweet, I appreciate it. 👍

**Comment by twilson63** [comment]
@grantsru 

Just published 1.0.1 version which should work with express v4, node v8+ and nano v7.

Thanks

Tom

**Comment by grantsru** [comment]
Fantastic! Thanks @twilson63
